CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # PowerShell Editor Services Release History
 
+## v3.3.1
+### Wednesday, April 20, 2022
+
+- 🐛 👷 [PowerShellEditorServices #1761](https://github.com/PowerShell/PowerShellEditorServices/pull/1766) - Bump `net461` to `net462` due to upcoming end of support.
+- 🐛 💎 [vscode-powershell #3928](https://github.com/PowerShell/PowerShellEditorServices/pull/1764) - Fix formatting handlers and PSScriptAnalyzer loading.
```

Comment:
is there supposed to be a vscode one in here?

Reply:
The script is built to put the bug number (if referenced in the PR) but link to the fix (the PR itself), so this is expected but looks funny.

Reply:
Ah I see that now, cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool cool

Reply:
GitHub suggested the extra cools

Reply:
😆